spec/lib/miq_automation_engine/miq_ae_service_model_spec.rb

```diff
@@ -136,4 +136,15 @@ module MiqAeServiceModelSpec
       end.to raise_error(NoMethodError)
     end
   end
+
+  describe MiqAeMethodService::MiqAeServiceStorageProfile do
+    let(:storage_profile) { FactoryGirl.create(:storage_profile) }
+    let(:ae_storage_profile) { MiqAeMethodService::MiqAeServiceStorageProfile.new(storage_profile.id) }
+
+    describe "#taggable" do
+      it "StorageProfile is taggable" do
+        expect(ae_storage_profile.taggable?).to be_truthy
```

Comment:
@jameswnl is this just testing that the model has `acts_as_miq_taggable`?  Not sure this tests anything specific to `StorageProfile`